keyboards/vertex/cycle8/info.json

```diff
@@ -0,0 +1,161 @@
+{
+    "keyboard_name": "Cycle8",
+    "manufacturer": "vertex",
+    "url": "",
+    "maintainer": "Eason",
+    "usb": {
+        "vid": "0x8A94",
+        "pid": "0x8F70",
+        "device_version": "0.0.1",
+        "force_nkro": true
+    },
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": true
+    },
+    "rgblight": {
+        "led_count": 20,
+        "animations": {
+            "breathing": true,
+            "rainbow_mood": true,
+            "rainbow_swirl": true,
+            "snake": true,
+            "knight": true,
+            "christmas": true,
+            "static_gradient": true,
+            "rgb_test": true,
+            "alternating": true,
+            "twinkle": true
+        }
+    },
+    "ws2812": {
+        "pin": "B15",
+        "driver": "spi"
+    },
+    "indicators": {
+        "caps_lock": "C13",
+        "scroll_lock": "A10"
+    },
+    "matrix_pins": {
+        "rows": ["B7", "B6", "B5", "B4", "B3","A15"],
+        "cols": [
+            "A1",
+            "A2",
+            "A3",
+            "A4",
+            "A5",
+            "A6",
+            "A7", "B0", "B1",
+            "B10",
+            "B11",
+            "B12",
+            "B13",
+            "B14",
+            "A8",
+            "A9"]
+    },
+    "diode_direction": "COL2ROW",
+    "processor": "STM32F103",
+    "bootloader": "uf2boot",
+    "layouts":{
+        "LAYOUT": {
```

Comment:
Quick check:  
Does this keyboard have more than one physical key layout?

Reason I ask: looks like there's an ISO Blackslash key in amongst an ANSI layout.

Reply:
I added physical layout of keys.

Reply:
> Quick check: Does this keyboard have more than one physical key layout?
> 
> Reason I ask: looks like there's ISO and JIS/ABNT2 keys yammed into an ANSI layout.

Hello, buddy. I don't know what's wrong, please check it for me. Thanks